README.md

```diff
@@ -204,3 +205,9 @@ pop up parachain -f ./tests/zombienet.toml -p https://github.com/r0gue-io/pop-no
 
 > :information_source: Pop CLI will automatically source the necessary polkadot binaries. Currently, these will be built
 > if on a non-linux system.
+
+## Acknowledgements
+
+Pop CLI would not be possible without these awesome crates!
```

Comment:
nit: Feels like just `zombienet-sdk` is the just the one "crate" getting credits among many others. Suggest this be reworded to say something akin to the help: "Deployment is powered by zombienet-sdk ..."

Reply:
Done